flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowDataSerDeSchemaTest.java

```diff
@@ -86,8 +89,9 @@ class JsonRowDataSerDeSchemaTest {
 
     private static final ObjectMapper OBJECT_MAPPER = JacksonMapperFactory.createObjectMapper();
 
-    @Test
-    void testSerDe() throws Exception {
+    @ParameterizedTest(name = "Is JsonParser: {0}")
```

Comment:
Why not use a class-level Parameterized test? You can take `RowLevelDeleteTest` as an example.